### DISCUSSION TITLE
Why am I being asked to justify my host permissions when my manifest doesn't have host permissions?

### DISCUSSION THREAD

**Discussion by Wizzel1** [wxt-dev/wxt#657] [Q&A] [answered]
This is the generated manifest from the `wxt build` step:
```json
{
  "manifest_version": 3,
  "name": "Uploader",
  "description":"",
  "version": "0.0.2",
  "icons": {
   ...
  },
  "homepage_url": "https://www.podsuite.io",
  "permissions": ["tabs", "identity", "storage"],
  "background": { "service_worker": "background.js" },
  "action": { "default_title": "PODSuite Uploader", "default_popup": "popup.html" },
  "content_scripts": [
    {
      "matches": ["*://*.merch.amazon.com/*"],
      "css": ["content-scripts/amazon.css"],
      "js": ["content-scripts/amazon.js"]
    },
    { "matches": ["https://www.google.com/"], "js": ["content-scripts/google.js"] }
  ]
}
```

and for some reason I need to give a host permission justification. Is that because of the content scripts?

![Unbenannt](https://github.com/wxt-dev/wxt/assets/43343510/63152767-1928-4437-9080-c6197ee1e036)

**Comment by aklinker1** [comment, marked as answer]
Yes, the content scripts. See the description to the right of the input. It mentions content scripts as a cause.

![Screenshot_20240508-152021.png](https://github.com/wxt-dev/wxt/assets/10101283/310ba740-04f2-4559-b9d0-e3d161ce9dd6)

**Reply by aklinker1** [reply]
Basically, they're just asking why you need content scripts

**Reply by Wizzel1** [reply]
@aklinker1 thanks I was just confused by an earlier answer on the discord channel where someone said that he never had to do this. Maybe I misunderstood something. https://discord.com/channels/1212416027611365476/1224505556346212515/1237833001673429053

**Reply by aklinker1** [reply]
Hmm, interesting... They could have been misremembering or misunderstanding. I've always had to add a reason for extensions with content scripts, don't think I've ever manually specified host permissions before.

**Reply by Wizzel1** [reply]
Anyways, thanks again for helping me out!
